pkgs/by-name/bk/BK7231GUIFlashTool/package.nix

```
@@ -0,0 +1,94 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  mono,
+  gtk2,
+  imagemagick,
+  makeWrapper,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
the point of finalAttrs is to replace rec

---

oh, i see!